src/Nethermind/Nethermind.Consensus/ConstantDifficultyCalculator.cs

```diff
@@ -24,6 +24,8 @@ public class ConstantDifficultyCalculator : IDifficultyCalculator
     {
```

Comment:
rename to ConstantDifficulty instead of ConstantDifficultyCalculator -> or even -> Difficulty.One, Difficulty.Zero